client/src/main/java/tw/waterball/survive/io/client/controller/PlayerController.java

```diff
@@ -2,13 +2,14 @@
 
 
 import tw.waterball.survive.io.api.api.ServerApi;
-import tw.waterball.survive.io.client.sprite.role.PlayerSprite;
-import tw.waterball.survive.io.client.sprite.weapon.WeaponSprite;
 import tw.waterball.survive.io.client.view.GameView;
+import tw.waterball.survive.io.game.sprite.role.Player;
+import tw.waterball.survive.io.game.sprite.weapon.Weapon;
+import tw.waterball.survive.io.game.world.SpriteContainer;
 
 public class PlayerController {
-    private PlayerSprite clientPlayerSprite;
-    private WeaponSprite playerCurrentWeaponSprite;
+    private Player clientPlayer;
+    private Weapon playerCurrentWeapon;
```

Comment:
Remove this line also, since we should get the current weapon from the player right?

Reply:
I fixed it. Thank you!